.env.development

```diff
@@ -1,11 +1,6 @@
 # APP VARIABLES - NEEDED FOR PRODUCTION
```

Comment:
Do we still need some `GITHUB_CLIENT_ID` somewhere like `.env` or `.env.local`? 
If so, we still need an .env.example right?

Reply:
yes we do.  I need to update the docs to reflect this new structure.  It would be under `.env.development.local`